language-files/yaml.md

```diff
@@ -33,24 +33,28 @@ Request support at <a href="mailto:i18n-ally@lokalise.com">i18n-ally@lokalise.co
 
 {% highlight yaml %}
 key:
-    - 'One value'
-    - 'Arrays are NOT supported'
+- 'One value'
+- 'Arrays are NOT supported'
 {% endhighlight %}
 
-### Translations nested under some key are not supported
+### Translations nested after a different key are not supported
+
+Nested translations are **only** supported if they start from the **root**, in the sample below, the translations are
+after a different key which is not supported.
 
 {% highlight yaml %}
-some_metadata:
-    ...
+metadata:
+    non_translation_key: true
 translations:
-    key: 'Key/values nested under some key are NOT supported'
+    key:
+        value: 'This is NOT supported, because i18n Ally calls it like `translations.key.value` while the app would use just `key.value`'
 {% endhighlight %}
 
 ### Complex structures within a key are not supported
 
 {% highlight yaml %}
 other_key:
-    translation: 'Complex structure per key is NOT supported'
-    notes: ''
+translation: 'Complex structure per key is NOT supported'
+notes: ''
```

Comment:
YAML syntax was broken here, you have to keep the indentation.

Reply:
Resolved with commit - 087da2713242d20b830302e7ab051078379a3e89